libs/fileset/file.go

```diff
@@ -7,7 +7,8 @@ import (
 
 type File struct {
 	fs.DirEntry
-	Absolute, Relative string
+	Absolute string `json:"absolute"`
+	Relative string `json:"relative"`
```

Comment:
Are these still needed? Looks like you have a dedicated type with JSON annotations under `bundle/deploy`.